clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -75,6 +74,7 @@
     private final int autoCommitIntervalMs;
     private final ConsumerInterceptors<?, ?> interceptors;
     private final boolean excludeInternalTopics;
+    private final AtomicInteger pendingAsyncCommits;
```

Comment:
Oh, I did miss one case. If we are closing while the coordinator is failing over. Hmm, in this case, we could still send any _unsent_ offset commit requests... I'm not sure if this case is worth trying to handle though.